users.md

```diff
@@ -1,6 +1,476 @@
 ---
 layout: contribute
-title: Bazel Users
+title: Who is using Bazel
 ---
 
```

Comment:
# Bazel community

In order to have a rendered page title, you need an H1 here. Perhaps we can change the title to "Bazel Community"? If not, let's keep the titles consistent, so please change this one & the title above to "Who's Using Bazel."
Thoughts?

Reply:
Sigh.  The two links pointing here used both wording. I made them "Who's using Bazel"

2 questions that I could not resolve from the style guide.
1. Do page titles and links follow the case-ing of headers: *Who's using Bazel* vs. *Who's Using Bazel*
2. Contractions. *Who's* vs *Who is*

I have been opting for no contractions on the grounds that they are harder for non English speakers to read.  I may be totally wrong on that. What is the Google wide thinking on that. My limited research on the internet says the debate is not a resolved one. 

LMK which works best:
1. submit this PR as is now and let you decide later
1. submit this PR as is now because it meets your intent, and you will update the style guide
1. change this to Who's Using (or Who is using?, Or Who is Using?) first, and then you will update the style guide

Reply:
I'm opting for choice 1, because we can fix capitalization later. Unblocking the Wiki deprecation is a more pressing goal at this time.